src/js/plugins/utils.js

```diff
@@ -29,6 +29,9 @@ export function mapPluginToCode(pluginUrl) {
         case 'gapro':
             code = 305006;
             break;
+        case 'bidding':
+            code = 305007;
+            break;
```

Comment:
This method should be stubbed in `jwplayer` and extended in jwplayer-commercial with this implementation containing commercial plugin error codes.

To extend this module, create src/js/plugins/utils.js in jwplayer-commmercial and import mapPluginToCode from os/plugins/utils.js.

Reply:
@robwalch All plugins in this `switch` are commercial only correct?

Reply:
Correct.